config/samples/perses.dev_v1alpha1_persesdashboard-2.yaml

```diff
@@ -9,555 +9,547 @@ metadata:
     app.kubernetes.io/created-by: perses-operator
   name: perses-dashboard-sample-2
 spec:
-  kind: Dashboard
-  metadata:
-    name: Demo-2
-    project: default
-    createdAt: "2021-11-09T00:00:00Z"
-    updatedAt: "2021-11-09T00:00:00Z"
-    version: 0
```

Comment:
I think this has not been updated after https://github.com/perses/perses-operator/commit/3a8e2193c0b03f2877286a01e64b391e5610cb2b. Now the resource is valid again. Same for the other dashboard resource.